doc/reference/reference_lua/box_info/synchro.rst

```diff
@@ -26,6 +26,8 @@ box.info.synchro
             To claim or reclaim the queue, use :ref:`box.ctl.promote() <box_ctl-promote>` on the instance that you want
             to promote.
             With elections enabled, an instance runs ``box.ctl.promote()`` command automatically after winning the elections.
+            To clear the ownership, call :ref:`box.ctl.demote() <box_ctl-demote>` on the instance that you want
+            to demote.
```

Comment:
Let's better say "on the synchro queue owner" rather than "on the instance you want to demote".

Reply:
Fixed